boa_engine/src/object/jsobject.rs

```diff
@@ -712,9 +740,9 @@ Cannot both specify accessors and a value or writable attribute",
 
     /// Inserts a field in the object `properties` without checking if it's writable.
     ///
-    /// If a field was already in the object with the same name that a `Some` is returned
-    /// with that field, otherwise None is returned.
-    pub fn insert_property<K, P>(&self, key: K, property: P) -> Option<PropertyDescriptor>
+    /// If a field was already in the object with the same name, than `true` is returned
+    /// with that field, otherwise `false` is returned.
```

Comment:
Hmm, here it says "`true` is returned with that field", but if `true` is returned, then the previous field is not returned, right?
This seems to be a leftover from the previous documentation comment, which should be updated.